2_run.R

```diff
@@ -1,37 +1,31 @@
 source('2_run/src/run_glm3.R')
-source('2_run/src/process_glm_output.R')
 
 p2 <- list(
   # Function will generate file
   # but return a tibble that includes that filename and its hash
   tar_target(
     p2_glm_uncalibrated_runs,
     run_glm3_model(
-      sim_dir = '2_run/tmp',
+      sim_dir = '2_run/tmp/simulations',
       nml_objs = p1_nml_objects,
       model_config = p1_model_config,
       burn_in = 300,
       burn_out = 190,
       export_fl_template = '2_run/tmp/GLM_%s_%s_%s.feather'),
     packages = c('retry','glmtools', 'GLM3r'),
     pattern = map(p1_model_config)),
+  
   # Group model runs by lake id and gcm
   # Discard the glm diagnostics so they don't trigger rebuilds
   # even when the export_fl_hash is unchanged
+  # Filter to only successful model runs within each group
```

Comment:
This comment makes it sound like you could still have groups which had some successful and some not. It's filtering to groups that had all successful runs so I suggest editing this comment:

```suggestion
  # Filter to groups where all model runs were successful (if any failed, the group is filtered out)
```